src/API/Router.ts

```diff
@@ -106,7 +106,12 @@ export class Router {
   }
 
   private postFile = async (context: KoaRouter.IRouterContext, next: () => Promise<any>) => {
+
     const files = context.request.files || {}
+
+    if (values(files).length <= 0)
+      context.throw(400, 'No file found.')
```

Comment:
Was thinking about unit testing this logic.

Reply:
@kennylavender just realized, I think this should be a 422, not a plain 400.

Reply:
@lautarodragan created an issue here: https://github.com/poetapp/node/issues/814